### PR TITLE
TUT-103 : Added login screen

### PR DESCRIPTION
**Pull request by preetipdapkekar**
- Modified activity_main.xml login screen
  Added linear layouts for user name and password
  Added login button
- Added dimen.xml file for dimentions
  Added dimentions for textfield
  Added margin for layout

### CODE REVIEW

**Comment by preetipdapkekar** [comment]
login_screen

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,18 +1,69 @@
...
+        <Button
+            android:id="@+id/login"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/login_layout_margin"
+            android:text="@string/login_text" />
+
+    </LinearLayout>
+
+
```
Comment (theyogeshrathod) [inline app/src/main/res/layout/activity_main.xml]: Remove this extra line.

File: app/src/main/res/values/dimen.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="login_layout_margin">20dp</dimen>
+    <dimen name="user_name">20sp</dimen>
```
Comment (theyogeshrathod) [inline app/src/main/res/values/dimen.xml]: Rename `user_name -> user_name_text_size`

File: app/src/main/res/values/dimen.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    <dimen name="password">20sp</dimen>
```
Comment (theyogeshrathod) [inline app/src/main/res/values/dimen.xml]: Rename `password  -> password _text_size`

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (preetipdapkekar) [reply app/src/main/res/layout/activity_main.xml]: okay ..I'll do.

File: app/src/main/res/values/dimen.xml (hunk 3 above)
Comment (preetipdapkekar) [reply app/src/main/res/values/dimen.xml]: ok. I will do all the changes.

**Comment by theyogeshrathod** [comment]
Please make the changes in the same branch:
Follow these steps:
- git checkout pr/preetipdapkekar/TUT-103-create-login-screen-byPreeti
- git pull
- Make changes in Android Studio
- git diff
- git add .
- git commit
- git push origin HEAD

Thank you.

**Comment by preetipdapkekar** [comment]
okay

**Comment by theyogeshrathod** [comment]
Thank you for the change. I'm merging this pull-request.

**Review by theyogeshrathod — APPROVED** [review]

**Pull request merged**